include/tvm/ffi/function.h

```diff
@@ -315,21 +315,21 @@ class Function : public ObjectRef {
    * \param packed_call The packed function signature
    */
   template <typename TCallable>
-  static Function FromPacked(TCallable packed_call) {
+  static Function FromPacked(TCallable&& packed_call) {
```

Comment:
Note: signature change. It's `std::forward`-ed in later use-sites, so I suppose this change should be fine

Reply:
This is indeed dangerous to forward expired pointers. Reverted